vol_test.h

```diff
@@ -207,6 +209,21 @@ extern char vol_test_filename[];
 
 #define VOL_TEST_FILENAME_MAX_LENGTH 1024
 
+#define VOL_TEST_PATH_MAX_LENGTH 1024
+
+/* Globally declare buffer name to let macro be used multiple times in same scope */
+//extern char *prefixed_filename_buffer;
+
+/* Prepend prefix to filename in dynamically allocated buffer, use user-declared pointer to return result. 
+ * User must free allocated memory after use. */
+#define PREFIX_FILENAME(buf, prefix, filename) \
+if (NULL == (buf = malloc(strlen(prefix) + strlen(filename) + 1))) {  \
+    fprintf(stderr, "can't allocate space for prefixed filename\n"); \
```

Comment:
`FUNC_GOTO_ERROR` macro isn't defined here, so I substituted this.

Reply:
The `TEST_ERROR` macro is available though, which is probably more appropriate. We should throw an error rather than just exiting.